test/optionsValidator.js

```diff
@@ -109,6 +109,22 @@ describe( 'options validator', function() {
 				} ).to.throw( 'Missing credential secret' );
 		} );
 
+        it( 'should set bucket to production cdn when none provided', function() {
+            var options = optionsValidator(
+                { id: "id", devTag: "devTag", creds: { key: "key", secret: "mySecret" } }
```

Comment:
Use `'` instead of '"'.

(I appreciate `"` is used elsewhere in this file, but that is not desired.)